packages/javascript/src/__tests__/index.test.ts

```diff
@@ -28,6 +28,44 @@ describe("Appsignal", () => {
     expect(result.namespace).toEqual("test")
   })
 
+  it("recieves an array of ignored patterns if one is passed to the constructor", () => {
+    const ignored = [/Ignore me/gm]
+
+    appsignal = new Appsignal({
+      key: "TESTKEY",
+      namespace: "test",
+      ignore: ignored
```

Comment:
Could we make the `ignore` key here `ignore_errors`? That would make it more consistent with the Ruby/Elixir API ( https://docs.appsignal.com/ruby/configuration/options.html#option-ignore_errors ) and leaves open the option to add an ignored actions/namespace feature in the future :)

Reply:
If we're keeping things consistent, should we also not make it string matching only? Not regex matching. In the Ruby & Elixir packages we match exact strings only.

Reply:
I have renamed `ignore` to `ignoreErrors`, but I will keep the RegExp matching the same as string matching doesn't need to be as useful given the way the `message` property is often used.

Reply:
We ignore errors through the agent in both Ruby and Elixir integrations, which is why we only allow strings for filters there. Is there any specific reason not to use the agent's filtering here?

Reply:
I'm not sure if I understand your question, this library doesn't use the agent @jeffkreeftmeijer so we do all the filtering in the library itself.

Reply:
> I will keep the RegExp matching the same as string matching doesn't seem to be as useful given the way the message property is often used.

Oh you're matching the message not the error type. That's definitely different than the other integrations. Is filtering by error type not useful here?

Reply:
@xadamy sorry, wrong library. Please carry on. :)

Reply:
@tombruijn Correct, error type names tend not to be as useful in JavaScript as the `Error` object is rarely subclassed.